rpm-manifest.json

```diff
@@ -0,0 +1,2119 @@
+{
```

Comment:
I don't think this file should be here. Please remove it.

Reply:
git add . ask questions later 😂

Reply:
removed 😅